src/components/MoneyRequestHeader.js

```diff
@@ -82,8 +83,12 @@ function MoneyRequestHeader({session, parentReport, report, parentReportAction,
 
     const isScanning = TransactionUtils.hasReceipt(transaction) && TransactionUtils.isReceiptBeingScanned(transaction);
     const isPending = TransactionUtils.isExpensifyCardTransaction(transaction) && TransactionUtils.isPending(transaction);
+    let canModifyRequest = isActionOwner && !isSettled && !isApproved && !ReportActionsUtils.isDeletedAction(parentReportAction);
 
-    const canModifyRequest = isActionOwner && !isSettled && !isApproved && !ReportActionsUtils.isDeletedAction(parentReportAction);
+    if (ReportUtils.isPaidGroupPolicyExpenseReport(moneyRequestReport)) {
+        // If it's a paid policy expense report, only allow modifying the request if it's not submitted or the user is the policy admin
+        canModifyRequest = canModifyRequest && (ReportUtils.isDraftExpenseReport(moneyRequestReport) || PolicyUtils.isPolicyAdmin(policy));
+    }
```

Comment:
I tested and found out that we can't add a receipt on a submitted expense too, so I decided to move the condition here.

But if the submitted expense has a receipt, we can replace/delete it.

Reply:
I tested and it seems that I can add a receipt on a submitted expense. Can you please double check?

Reply:
Let me check once again.

Reply:
Yeah, I can add a receipt on a submitted request. Updated